Tests/Tracking/CustomerIOImplementationTest.swift

```diff
@@ -129,6 +129,32 @@ class CustomerIOImplementationTest: UnitTest {
         XCTAssertEqual(profileIdentifyHookMock.profileIdentifiedCallsCount, 1)
     }
 
+    func test_identify_givenEmptyIdentifier_givenNoProfilePreviouslyIdentified_expectDoNotRunHooks() {
```

Comment:
```suggestion
    func test_identify_givenEmptyIdentifier_givenNoProfilePreviouslyIdentified_expectRequestIgnored() {
```

We want to check for more then just hooks executing (example: `XCTAssertNil(profileStoreMock.identifier)` is a great test) so I suggest a function name change.